src/interfaces/records/IItemRecord.ts

```diff
@@ -1,7 +1,13 @@
 export interface IItemRecord {
+  nodes: Object;
   id: string;
   name: string;
   description: string;
   versionTag: string;
   active: boolean;
+  historyTimeToLive: string;
+  deploymentId: string;
+  initialContext: Object;
+  setup: Object;
+  _links: Object;
```

Comment:
This is in use by the table list and it does not load all of these.

Make all of the added properties optional, such as ` nodes?: Object;`.

The types only happen to be on the list at the time because of a bug.
In the future, all of the ones you added here will not be present.

An additional resource load is (or will be) necessary to load all of the remaining parts.